data/content/environment_variables.yaml

```diff
@@ -97,17 +97,29 @@ variables:
   example: "cdanvers@kree-net.com"
 - name: BUILDKITE_BUILD_CREATOR
   desc: |
-    The name of the user who created the build. May be **[unverified](#unverified-commits)**.
+    The name of the user who created the build. The value differs depending on how the build was created:
+      
+    - **Buildkite dashboard:** Set based on who manually created the build.
+    - **GitHub webhook:** Set from the  **[unverified](#unverified-commits)** HEAD commit.
+    - **Webhook:** Set based on which user is attached to the API Key used.
   modifiable: false
   example: "Carol Danvers"
 - name: BUILDKITE_BUILD_CREATOR_EMAIL
   desc: |
-    The notification email of the user who created the build.
+    The notification email of the user who created the build. The value differs depending on how the build was created:
+
+    - **Buildkite dashboard:** Set based on who manually created the build.
+    - **GitHub webhook:** Set from the  **[unverified](#unverified-commits)** HEAD commit.
+    - **Webhook:** Set based on which user is attached to the API Key used.
   modifiable: false
   example: "cdanvers@kree-net.com"
 - name: BUILDKITE_BUILD_CREATOR_TEAMS
   desc: |
-    A colon separated list of **[unverified](#unverified-commits)** non-private team slugs that the build creator belongs to.
+    A colon separated list of non-private team slugs that the build creator belongs to. The value differs depending on how the build was created:
+
+    - **Buildkite dashboard:** Set based on who manually created the build.
+    - **GitHub webhook:** Set from the  **[unverified](#unverified-commits)** HEAD commit.
+    - **Webhook:** Set based on which user is attached to the API Key used.
```

Comment:
@dannymidnight, some unexpected behavior with the preview for this PR. You can see the formatting for this entry is the same as the two above, but it doesn't display right on the [preview](https://2182--bk-docs-preview.netlify.app/docs/pipelines/environment-variables):

![Screenshot 2023-06-07 at 9 55 28 am](https://github.com/buildkite/docs/assets/119824349/37a2f631-b520-440f-86ee-1790065de0d0)

When I run it locally, it shows as expected.